OpenLaTeXStudio/EditorModule/src/latexstudio/editor/ButtonColumn.java

```diff
@@ -0,0 +1,240 @@
+package latexstudio.editor;

```

Comment:
Can we move it to a new package, to distinguish that it is an external solution? The package could be named lib.